dockerfiles/testsuite/start_test.sh

```diff
@@ -116,14 +117,18 @@ e2etest_tokentxs() {
 	e2etest tokentxs
 }
 
-e2etest_overwritelection () {
+e2etest_overwritelection() {
   e2etest overwritelection
 }
 
-e2etest_cenususizelection () {
+e2etest_cenususizelection() {
```

Comment:
another candidate to be moved into "e2etest small fixes"
cenususizelection -> censusizelection

Reply:
Yeah, sure I will create a small PR for those